api/src/shared/infrastructure/repositories/user-login-repository.js

```diff
@@ -20,6 +21,15 @@ const findByUserId = async function (userId) {
   return foundUserLogin ? _toDomain(foundUserLogin) : null;
 };
 
+const getByUserId = async function (userId) {
+  const foundUserLogin = await findByUserId(userId);
+  if (!foundUserLogin) {
+    throw new NotFoundError();
+  }
+
+  return foundUserLogin;
```

Comment:
Suggestion :  
Faire le mapping vers un objet du domaine au lieu de retourner, directement le résultat de la base de donnée 😸 
```suggestion
  return _toDomain(foundUserLogin);
```

Reply:
`getByUserId` récupère le `foundUserLogin` depuis `findByUserId` qui fait bien un `_toDomain`, donc c'est bon :-) 

Par contre on a dans les fonctions `findByUserId` et `getByUserId` une variable `foundUserLogin` qui porte le même nom mais ne correspond pas à la même chose. C'est à modifier pour que le fonctionnement soit plus clair. Je vais ajouter un commit dans ce sens !

Reply:
:heavy_check_mark: Voilà je viens d'ajouter un nouveau commit renommant les variables de DTO comme telles. Cela permet de différencier clairement les _DTO_ des _Domain Objects_.

Reply:
merci !